torch/csrc/api/src/nn/modules/embedding.cpp

```diff
@@ -9,40 +9,44 @@
 #include <utility>
 #include <vector>
 
+namespace F = torch::nn::functional;
+
 namespace torch {
 namespace nn {
 EmbeddingImpl::EmbeddingImpl(const EmbeddingOptions& options_) : options(options_) { // NOLINT(modernize-pass-by-value)
   reset();
 }
 
 void EmbeddingImpl::reset() {
+  TORCH_CHECK(*options.num_embeddings(), "num_embeddings needs to be defined");
+  TORCH_CHECK(*options.embedding_dim(), "embedding_dim needs to be defined");
```

Comment:
good catch! I intended to write options.num_embeddings(). I think there will be a memory access error in case they are not defined because we'll be dereferencing a NULL pointer.